lib/src/tokenizer.dart

```diff
@@ -570,7 +570,8 @@ class HtmlTokenizer implements Iterator<Token> {
     } else if (data == ">") {
       emitCurrentToken();
     } else if (data == EOF) {
-      _addToken(new ParseErrorToken("eof-in-tag-name"));
+      _addToken(new CharactersToken("<" + currentTagToken.name));
```

Comment:
this change is not spec compliant. this should be an error, see: https://html.spec.whatwg.org/multipage/syntax.html#tag-name-state